modules/administration/pages/image-management.adoc

```diff
@@ -187,46 +162,39 @@ If your container image tag is in a format such as `myproject/myimage`, make sur
 ----
 https://github.com/USER/project.git#branchname:folder
 ----
-
++
```

Comment:
I think here, for example, the `+` connector is not needed.  But I do not want to create a test build now.